### PR TITLE
Add css.at-rules.font-face.font-stretch

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
A part of https://github.com/mdn/sprints/issues/1378 to add basic BCD.  Values were identified via manual testing within SauceLabs.

### CODE REVIEW

File: css/at-rules/font-face.json [hunk 1]
```diff
@@ -463,6 +463,57 @@
...
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": null
+              },
+              "firefox_android": {
+                "version_added": null
+              },
+              "ie": {
+                "version_added": null
```
Comment (Elchi3) [inline css/at-rules/font-face.json]: If this gets merged, it would cause a failing build as we now disallow `null` values for IE.

**Comment by ddbeck** [comment]
@vinyldarkscratch do you plan to revisit this one?

**Comment by vinyldarkscratch** [comment]
I actually totally forgot about this particular PR!  I do -- I'll write up a test for IE and check it out in SauceLabs within the next 48 hours.  :wink:

**Comment by vinyldarkscratch** [comment]
Looks like IE doesn't support `font-stretch`.  Marked as false!

File: css/at-rules/font-face.json [hunk 2]
```diff
@@ -463,6 +463,57 @@
...
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": "11.1"
```
[the remark this answers is not in the document]
Comment (vinyldarkscratch) [reply css/at-rules/font-face.json]: That seems reasonable to me, 10.1 it is!

**Comment by vinyldarkscratch** [comment]
Based upon manual testing, I was able to determine that support was first implemented in Chrome 62.  So, that finishes off our compatibility data for this feature!

**Pull request merged**